public/index.js

```diff
@@ -0,0 +1,86 @@
+let myMap = L.map('mainMap').setView([44.475883, -73.212074], 13);
+
+L.tileLayer('https://tiles.stadiamaps.com/tiles/outdoors/{z}/{x}/{y}{r}.png', {
+    maxZoom: 20,
+    attribution: '&copy; <a href="https://stadiamaps.com/">Stadia Maps</a>, &copy; <a href="https://openmaptiles.org/">OpenMapTiles</a> &copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributors'
+}).addTo(myMap)
+
+//****************resto-list code ****************/
+let listContainer = document.getElementById('restoList')
+
+async function getPosts() {
+    let postList = await fetch('/api/restaurants.json')
```

Comment:
To get the postList from your `all.json` file for the Rutland data you can use this same fetch except fetch `/api/all.json` instead if you set up your server as recommended above.